sources/packages/web/src/components/students/modals/ConfirmEditApplication.vue

```diff
@@ -6,11 +6,37 @@
       sure you want to proceed?
     </template>
     <template #content v-else>
-      Any edits made to your application may result in a new assessment,
-      potentially delaying your application. Are you sure you want to proceed?
+      <v-checkbox
+        v-model="conditionsAccepted"
+        label="I acknowledge that by editing my application, I may be required to
+        resubmit previously approved exception requests. My institution may be
+        required to resubmit program information, and my parent or partner may be
+        required to resubmit supporting information."
+        hide-details="auto"
+      ></v-checkbox>
+      <banner
+        class="mb-4"
+        v-if="endDate !== '' && dayjs(endDate).diff(dayjs(), 'day') < 42"
```

Comment:
The logic to show the banner must be the 6 week deadline policy from the ticket [2997](https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/2997) 

The indicators are available in student application through 2 hidden variables. If one of them id true, that is when the banner must be displayed.

Reply:
I am able see 2 possible solutions here. 

Background:  `wizardSubmit()` does not have access to the form submitted data. 

1. In the component `StudentApplication.vue` the function `formChanged()` can emit the result of the variables `studyEndDateBeforeSixWeeksFromToday || selectedStudyEndDateBeforeSixWeeksFromToday` to parent `FullTimeApplication.vue` and then `FullTimeApplication.vue`  can use this value to show banner.
2.  When the submit button is clicked in `FullTimeApplication.vue` then the logic to show modal (Confirm Modal) can be moved into `submitApplication()` so that it can check these formio values and set the show banner indicator.

Reply:
@dheepak-aot maybe we can get the value from the hidden using the formioUtils `getComponent`. Does it make sense?

Reply:
With my recent understanding, I would suggest the following way to use the highlighted variable to show the banner. 

![image](https://github.com/user-attachments/assets/1be5cf6b-266f-44bf-9275-6c5ef9cda228)